frontend/src/components/navbar/Navbar.tsx

```diff
@@ -15,17 +22,34 @@ export const Navbar: React.FunctionComponent = () => {
           alt="Whale Spotting logo"
         />
       </Link>
-      <div>
+      <span className={`menu-items ${menuCollapsed && "menu-collapsed"}`}>
+        <Link className="menu-link" to="/report-sighting">
+          Report Sighting
+        </Link>
+        <Link className="menu-link" to="/ticket-booking">
+          Book Tickets
+        </Link>
+        <Link className="menu-link" to="/sightings-list">
+          Sightings List
+        </Link>
```

Comment:
Links shouldn't exist if they don't yet lead anywhere! Don't add them just to be placeholders